model/src/main/kotlin/ProjectAnalyzerResult.kt

```diff
@@ -39,6 +41,7 @@ data class ProjectAnalyzerResult(
         /**
          * The list of errors that occurred during dependency resolution. Defaults to an empty list.
          */
+        @JsonInclude(JsonInclude.Include.NON_EMPTY)
```

Comment:
I believe we're never ever (de-)serializing this class directly, so we *could* drop this.

Reply:
It's deserialized in some tests, so I liked to have it here for consistency.

Reply:
Maybe more concise: "Do not serialize if empty for consistency with the error properties in other classes, even if this class is not serialized as part of an [OrtResult]."